spec/system/move_tokens_spec.rb

```diff
@@ -4,17 +4,19 @@
   it "shows tokens on the map" do
     map = create :map
     token = create :token, map: map
-    visit campaign_path(map.campaign)
+    visit campaign_path(map.campaign, as: map.campaign.user)
+    wait_for_connection
```

Comment:
Not a note for this PR, but I wonder if there's a generic way we can get Capybara to always wait for connections if one is being established

Reply:
Good catch, at one point I was getting unexpected failures and started overzealously adding it.